src/data/contributors/contributors.json

```diff
@@ -0,0 +1,115 @@
+{
+  "projectName": "website",
+  "projectOwner": "openclimatefix",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": true,
+  "commitConvention": "gitmoji",
+  "contributors": [
+    {
+      "login": "JackKelly",
+      "name": "Jack Kelly",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/460756?v=4",
+      "profile": "http://jack-kelly.com",
+      "contributions": [
+        "business",
+        "code",
+        "content",
+        "ideas"
+      ]
+    },
+    {
+      "login": "FWirtz",
+      "name": "Flo",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/6052785?v=4",
+      "profile": "https://github.com/FWirtz",
+      "contributions": [
+        "code",
+        "ideas",
+        "maintenance",
+        "projectManagement"
+      ]
+    },
+    {
+      "login": "dctanner",
+      "name": "Damien Taner",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/227?v=4",
+      "profile": "http://tanner.me",
+      "contributions": [
+        "blog"
+      ]
+    },
+    {
+      "login": "schillerk",
+      "name": "Kyle Schiller",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/8676510?v=4",
+      "profile": "https://github.com/schillerk",
+      "contributions": [
+        "code",
+        "design"
+      ]
+    },
+    {
+      "login": "ollicle",
+      "name": "Oliver Boermans",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/63586?v=4",
+      "profile": "http://www.ollicle.com",
+      "contributions": [
+        "content",
+        "bug",
+        "ideas"
+      ]
+    },
+    {
+      "login": "eambutu",
+      "name": "eambutu",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/11009767?v=4",
+      "profile": "https://www.phillipkwang.com",
+      "contributions": [
+        "bug",
+        "code"
+      ]
+    },
+    {
+      "login": "openclimatefix",
+      "name": "RAAIS Foundation",
```

Comment:
let's remove RAAIS from this one as we already have them under "Funding" on the website 😊